crates/wallet/src/wallet/mod.rs

```diff
@@ -1704,7 +1706,9 @@ impl Wallet {
             .iter()
             .any(|txin| txin.sequence.to_consensus_u32() <= 0xFFFFFFFD)
         {
-            return Err(BuildFeeBumpError::IrreplaceableTransaction(tx.txid()));
+            return Err(BuildFeeBumpError::IrreplaceableTransaction(
+                tx.compute_txid(),
```

Comment:
Also, L1671 there's an old `.txid()` in a doc comment

Reply:
A few other spots in doc code that need to have tx.txid() fixed:
chain, tx_data_traits.rs: L46, L61, L79
wallet, wallet/mod.rs: L1671

Reply:
Thanks, sorry for the oversight, it's probably because `rust-analyzer` does not handle these due to being under feature/docs codes, and it didn't show up as failure. I'll handle these.